examples/pytorch/graphsage/train_full.py

```diff
@@ -111,40 +38,29 @@ def main(args):
                       args.n_layers,
                       F.relu,
                       args.dropout,
-                      args.aggregator_type)
-
-    if cuda:
-        model.cuda()
-
+                      args.aggregator_type).to(device)
     # use optimizer
     optimizer = torch.optim.Adam(model.parameters(), lr=args.lr, weight_decay=args.weight_decay)
 
     # initialize graph
     dur = []
     for epoch in range(args.n_epochs):
         model.train()
-        if epoch >= 3:
-            t0 = time.time()
+        t0 = time.time()
```

Comment:
The speed becomes more stable after first running for a few epochs, though I'm not sure if it is necessary to include timing logic in the example code. @jermainewang

Reply:
From my point of view, it doesn't hurt reporting elapse time per epoch. What I don't quite understand is, originally (1) why we report average time for each epoch (that seems not quite intuitive and informative to me....) (2) what is ETputs? edge throughput? Is it a well-acknowledged quantity to quantify speed as (# of edge/ms)? If not, I think we should remove it cuz it's confusing.  Also, I think it makes more sense to use the actual elapse epoch time to quantify "speed" though....

I slightly update the logging format to have per epoch elapse time and total training time as :
```
Epoch xxx | Time(s) 0.0058 | Loss 0.0875 | Accuracy 0.6480 | ETputs(KTEPS) 1579.65
Training time(s) 2.5515

Test Accuracy 0.6760
```

In this way, the first few epoch slow-down is very clear in log.

Reply:
(1) I don't think reporting training time report is very necessary. Users can always add that themselves.
(2) You could remove the reporting of ETputs (edge training throughput).